src/homework/day8/Noodle.java

```diff
@@ -0,0 +1,40 @@
+package homework.day8;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Noodle {
+    public static void main(String[] args) {
+        List<String> noodles = new ArrayList<>();
+
+        noodles.add("Hakka");
+        noodles.add("Ramen");
+        noodles.add("Hibachi");
+        noodles.add("Schezwan");
+
+//Проитерировать список через for-each и отпечатать слова в консоль через тире
+
+        System.out.println("Task1");
+        for (String noodle : noodles) {
+            System.out.print(noodle + "-");
```

Comment:
Here as a result you will have "-" after the last printed noodle, I guess that better to exclude it